asd-aituber/apps/web/__tests__/performance-monitoring-basic.test.ts

```
@@ -0,0 +1,113 @@
+  },
+  now: vi.fn(() => Date.now()),
+  mark: vi.fn(),
+  measure: vi.fn(),
+} as any
+
+global.navigator = {
+  userAgent: 'test-user-agent'
+} as any
+
+class MockPerformanceObserver {
+  constructor(callback: PerformanceObserverCallback) {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)